### PR TITLE
[WIP]fix ci

### PR DESCRIPTION
**Pull request by hwdef**

### CODE REVIEW

File: pkg/config/config_test.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 package config
 
+// aaa
```
Comment (dongjiang1989) [inline pkg/config/config_test.go:19]:
```suggestion
```

Comment (dongjiang1989) [reply pkg/config/config_test.go:19]: Invalid comment

Comment (hwdef) [reply pkg/config/config_test.go:19]: This is only temporary content. And this PR is WIP

Comment (dongjiang1989) [reply pkg/config/config_test.go:19]: > This is only temporary content. And this PR is WIP

I got it. thanks

**Pull request closed without merging**